### PR TITLE
Challenge1

### PR DESCRIPTION
**Pull request by sambiase**
new request

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,10 +8,20 @@
...
 
+    <dependencies>
+        <!-- https://mvnrepository.com/artifact/org.apache.commons/commons-csv -->
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-csv</artifactId>
+            <version>1.8</version>
+        </dependency>
+    </dependencies>
+
```
Comment (jcralmeida) [inline pom.xml:19]: Dependencies go after the properties

File: src/main/java/readCSV.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+
+public class readCSV {
+
+    static final String READ_FILE = "/home/sambiase/JavaProjects/desafios_java/desafios_java/src/main/java/names.csv";
+    static final String WRITE_FILE ="/home/sambiase/JavaProjects/desafios_java/desafios_java/src/main/java/times.csv";
+    public static void main(String[] args) {
```
Comment (jcralmeida) [inline src/main/java/readCSV.java:14]: Let's create a new file called Main, add move the main method to it

File: src/main/java/readCSV.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import org.apache.commons.csv.CSVRecord;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+
+public class readCSV {
+
+    static final String READ_FILE = "/home/sambiase/JavaProjects/desafios_java/desafios_java/src/main/java/names.csv";
+    static final String WRITE_FILE ="/home/sambiase/JavaProjects/desafios_java/desafios_java/src/main/java/times.csv";
```
Comment (jcralmeida) [inline src/main/java/readCSV.java:13]: Let's receive this values as program arguemnts

Comment (jcralmeida) [reply src/main/java/readCSV.java:13]: Also this values should be obtained via Main method that you should create

File: src/main/java/readCSV.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    public static void main(String[] args) {
+
+        String [] times = {"Mengão","Fluminense", "Botafogo", "Vasco"};
+
+        // READ CSV
+        try {
+            CSVParser csvRead = new CSVParser(new FileReader(READ_FILE),CSVFormat.DEFAULT.withHeader("id","first_name","last_name"));
```
Comment (jcralmeida) [inline src/main/java/readCSV.java:20]: Not a good practice to hard-coded the schema

File: src/main/java/readCSV.java [hunk 5]
```diff
@@ -0,0 +1,43 @@
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVPrinter;
+import org.apache.commons.csv.CSVRecord;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+
+public class readCSV {
```
Comment (jcralmeida) [inline src/main/java/readCSV.java:10]: Let's change this name since it does not only read csv but it writes too

**Review by jcralmeida — CHANGES_REQUESTED** [review]

File: src/main/java/readCSV.java (hunk 2 above)
Comment (sambiase) [reply src/main/java/readCSV.java]: Done

File: src/main/java/readCSV.java (hunk 3 above)
Comment (sambiase) [reply src/main/java/readCSV.java]: Implemented as requested

File: src/main/java/readCSV.java (hunk 4 above)
Comment (sambiase) [reply src/main/java/readCSV.java]: resolved

**Pull request closed without merging**